GETTING_STARTED.md

```diff
@@ -64,4 +64,42 @@ Click on `Issues` to find your first Week in the camp: `Getting Started`.
 
 **Click** on `Getting Started` to find your first step of the week. **Follow your counselor's** instructions and have fun!
 
-![image](https://user-images.githubusercontent.com/69332964/114479022-8fe9d780-9bcd-11eb-956f-6e4132220e11.png)
\ No newline at end of file
+![image](https://user-images.githubusercontent.com/69332964/114479022-8fe9d780-9bcd-11eb-956f-6e4132220e11.png)
+
+### :x: The Step Failed. What do I do?
+No worries, we all make mistakes - now is the time to fix it! The CounselorBot will provide feedback on what went wrong with your code. Here's how to see it:
+
+First, click on `Actions` at the top of your repository.
+![image](https://user-images.githubusercontent.com/69332964/120932906-b4da6280-c6c5-11eb-97f8-b8311c72eabb.png)
+
+Next, find the commit that failed. You can identify it by the name you called the commit. **Click on it!**
+<img width="1019" alt="Screen Shot 2021-06-06 at 12 51 31 PM" src="https://user-images.githubusercontent.com/69332964/120933012-3cc06c80-c6c6-11eb-9ca9-f3b19a8bb0b8.png">
+
+Now, go ahead and click on ":x: build"
+![image](https://user-images.githubusercontent.com/69332964/120933055-68435700-c6c6-11eb-88a9-af248114c6a9.png)
+
+The step that failed **should expand** and you'll see feedback at the bottom. 
+![image](https://user-images.githubusercontent.com/69332964/120933130-a9d40200-c6c6-11eb-91a6-c6a477aa62dc.png)
+
+For this one, the feedback was:
+```
+Try again!
+We go letmein and incorrect, but it should be "Access granted." and "Access denied."
```

Comment:
I suggest to use the improved message here: 
`We submitted "letmein" and got ${correct}, which should equal "Access granted."`

Reply:
@marcduiker Sounds good! I did change the error message on the test, but not on the instructions. Thanks 👍